### ISSUE TITLE
WPF Axis/Tracker LabelFormatter

### ISSUE THREAD

**Issue by Maxima078** [oxyplot/oxyplot#586]
Hello,

Is there a way to differentiate the LabelFormatter from Axis and Series tracker?
When I set LabelFormatter in CategoryAxis I wasn't expecting the same Formatter will be used in series tracker.

Is this a normal behavior?
In the tracker I have enough space to display the label entirely, but below my axis I have not. This is why I would like to use different formatter....

Thanks.

**Comment by objorke** [comment]
I agree this should be improved. Create a new property `TrackerLabelFormatter`?

**Comment by forum111** [comment]
lineseries1.TrackerLabelFormat = ....
